src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -53,14 +53,14 @@ Add your Dart code to the `lib/` directory.
 Add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter packages
 and plugins.
 
-The `.ios/` hidden subfolder contains a Xcode workspace where you can
-run a stand-alone version of your module, and is a wrapper project to bootstrap
-your Flutter code. It contains helper scripts to build frameworks or
+The `.ios/` hidden subfolder contains an Xcode workspace where you can
+run a standalone version of your module, and contains a wrapper project to bootstrap
```

Comment:
The subject of "is a wrapper project" here was intended to be the Xcode workspace, not the `.ios` directory, so it doesn't make sense to say it _contains_ a wrapper project when it _is_ the wrapper project.  Maybe just make it's clearer as a new sentence?
> It is a wrapper project to bootstrap your Flutter code.